sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/Helpers/UcfgSerializerTest.cs

```diff
@@ -59,9 +59,9 @@ void Foo(string a, string b)
 0 [shape=record label=""{BLOCK:0|%0 __concat b,a|x __id %0|TERMINATOR JUMP: 1, 2}""]
 0 -> 1
 0 -> 2
-1 [shape=record label=""{BLOCK:1|%0 C.Bar(string, int) a,CONST|TERMINATOR JUMP: 3}""]
+1 [shape=record label=""{BLOCK:1|%1 C.Bar(string, int) a,CONST|TERMINATOR JUMP: 3}""]
```

Comment:
Why this change?

Reply:
The temp variables do not start from %0 for each block, the numbering is shared for the whole UCFG.

Reply:
Because the UcfgObjectFactory (which contains the temp variable counter) is not created for each block, but shared for all blocks.

Reply:
It used to start at 0, why this change? Is it not going to "break" something?

Reply:
It used to create new InstructionBuilder (the old name) for each block. Now the instruction generation is shared for the whole UCFG. It could be like before, but there is no need.